common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -97,6 +97,7 @@ def test_student_view_init(self):
     def test_render_student_view(self):
         html = self._get_rendered_student_view(
             self.sequence_3_1,
+            requested_child='first',
```

Comment:
This test was failing/passing with the exact same code, I suspect it was due to a prior test requesting a different child and that child being "saved" as the current location before running this test. Does this change invalidate the assumptions under test, or is everyone okay with this?